### PR TITLE
Make sensitive columns optional

### LINKED ISSUE #55: Als record manager wil ik dat er een verklaring van vernietiging (of proces verbaal) wordt gegenereerd over de vernietiging van documenten
zodat ik kan voldoen aan de archiefwet. De verklaring van vernietiging (of proces verbaal) is het bewijs van daadwerkelijke vernietiging. Hierin is opgenomen: een specificatie van het vernietigde (de vernietigingslijst), op grond waarvan vernietigd is (de selectiecategorie) en de wijze waarop vernietigd is (een beschrijving hoe technisch is vernietigd; hierbij is het van belang duidelijk te maken hoe ervoor is gezorgd dat onherstelbaar is vernietigd).   
graag oppakken met #62 

[Thema uit conceptofferte: 2. Proces-verbaal/rapport genereren van vernietiging] 

--

*Clearification*

There should be like a template document with a table as attachment. This template document should be configuratble. Columns for the attachment are:

image

* SAD should be gemeentearchivaris.

*Guestimate*

* Uniek kenmerk = Zaak.identificatie
* Beschrijving = Zaak.omschrijving
* Looptijd = Zaak.Einddatum - Zaak.Startdatum
* VCS = Zaak.Zaaktype.Selectielijstprocestype.Nummer
* Toelichting = App.DestructionList.toelichting
* Opmerkingen = App.DestructionList.comments
* Reactie zorgdrager = Comment from process owner

### PR DESCRIPTION
**Pull request by SilviaAmAm**
Fixes #55 

**What's new**
- The columns Beschrijving and Opmerkingen are optional
- The destruction list has a new field to specify whether it contains privacy-sensitive information
- The destruction list creation form was updated so that the record manager can specify whether a list contains sensitive data

*Note*
There is a column title missing in the report, but this is fixed in https://github.com/maykinmedia/archiefvernietigingscomponent/pull/150

=> Needs to be rebased after https://github.com/maykinmedia/archiefvernietigingscomponent/pull/146 is merged

### CODE REVIEW

**Comment by annashamray** [comment]
@SilviaAmAm Could you please rebase this PR on the master branch? I'll review it after rebase

File: src/archiefvernietigingscomponent/destruction/forms.py [hunk 1]
```diff
@@ -52,10 +52,17 @@ class DestructionListForm(forms.ModelForm):
     reviewer_2 = forms.ModelChoiceField(
         queryset=User.objects.reviewers().all(), required=False
     )
+    contains_sensitive_info = forms.BooleanField(required=False)
```
Comment (annashamray) [inline src/archiefvernietigingscomponent/destruction/forms.py:55]: I don't think this line is necessary, since `contains_sensitive_info` is a model field

**Review by annashamray — APPROVED** [review]
Looks very nice!

File: src/archiefvernietigingscomponent/destruction/forms.py (hunk 1 above)
Comment (SilviaAmAm) [reply src/archiefvernietigingscomponent/destruction/forms.py]: Removed it!

**Pull request merged**